stem/kratos_IO.py

```diff
@@ -47,16 +52,78 @@ def __write_output_processes(self):
     def __write_input_processes(self):
         pass
 
+    def __write_constraints(self):
+        pass
+
+    def __write_loads(self):
+        pass
+
+    def __create_load_dict(self, load: Load) -> Dict[str, Any]:
+        """
+        Creates a dictionary containing the load parameters
+        Args:
+            load (Load): load object
+        Returns:  Dict[str, Any]: dictionary containing the load parameters
+        """
+
+        # initialize load dictionary
+        load_dict: Dict[str, Any] = {
+            "python_module": None,
+            "kratos_module": None,
+            "process_name": None,
+            "Parameters": {},
+        }
+
+        # add load parameters to dictionary based on load type.
+        if isinstance(load.load_parameters, PointLoad):
+            load_dict.update(
```

Comment:
dictionary.update, is only required if you extend the dictionary with new keys. In this case you can do load_dict["python_module"] =  "apply_vector_constraint_table_process"